ui/src/clientLibraries/graphics/JSLogo.tsx

```diff
@@ -17,12 +17,12 @@ const JSLogo: SFC = () => {
            .js2{fill-rule:evenodd;clip-rule:evenodd;fill:#FFFFFF;}
            .js3{fill-rule:evenodd;clip-rule:evenodd;fill:#699F63;}
            .js4{clip-path:url(#XMLID_5_);}
-           .js5{fill:url(#SVGID_1_);}
-           .js6{fill:url(#SVGID_2_);}
-           .js7{fill:url(#SVGID_3_);}
-           .js8{fill:url(#SVGID_4_);}
-           .js9{fill:url(#SVGID_5_);}
-           .js10{fill:url(#SVGID_6_);}
+           .js5{fill:#699F63;}
+           .js6{fill:#699F63;}
```

Comment:
Can we move these to a stylesheet?

Reply:
this matches the format of our other logo files.

Reply:
True, but we can also make things a little clearer if we wanted to. It's definitely not a deal breaker but something to keep in mind since we are doing it here: